delocate/fuse.py

```diff
@@ -68,8 +68,8 @@ def _retag_wheel(to_wheel: Path, from_wheel: Path, to_tree: Path) -> str:
     to_tree = to_tree.resolve()
     # Add from_wheel platform tags onto to_wheel filename, but make sure to not
     # add a tag if it is already there
-    from_wheel_tags = parse_wheel_filename(from_wheel.name)[-1]
-    to_wheel_tags = parse_wheel_filename(to_wheel.name)[-1]
+    _, _, _, from_wheel_tags = parse_wheel_filename(from_wheel.name)
```

Comment:
How about:
```suggestion
    *_, from_wheel_tags = parse_wheel_filename(from_wheel.name)
```

Reply:
I prefer the explicit syntax over tuple repacking or last item index since this clarifies the size of the tuple being unpacked and which items are being unpacked onto which names as well as making clear that these items are distinct from each other and that this isn't just a plain list with the last item being accessed.

Your suggestion is equivalent to a more complex last index access, in which case I should've just kept these lines unchanged.

A custom NamedTuple could also work here. Anything to clarify that the return value is a structured tuple.